src/doc/unstable-book/src/library-features/asm.md

```diff
@@ -621,7 +621,9 @@ Some registers cannot be used for input or output operands:
 | MIPS32 | `$0` or `$zero` | This is a constant zero register which can't be modified. |
 | MIPS32 | `$1` or `$at` | Reserved for assembler. |
 | MIPS32 | `$26`/`$k0`, `$27`/`$k1` | OS-reserved registers. |
-| MIPS32 | `$28`/`$gp`, `$29`/`$sp`, `$30`/`$s8`/`$fp` | Global pointer, stack pointer, and frame pointer cannot e used as inputs or outputs. |
+| MIPS32 | `$28`/`$gp` | Global pointer cannot be used as inputs or outputs. |
+| MIPS32 | `$29`/`$sp` | Stack pointer cannot be used as inputs or outputs. |
+| MIPS32 | `$30`/`$s8`/`$fp` | Frame pointer cannot be used as inputs or outputs. |
```

Comment:
The stack pointer and frame pointer are already covered by the first two lines of the table. You don't need to add MIPS-specific entries for them.